ontology/observable/observable.ttl

```diff
@@ -3724,7 +3730,7 @@ observable:MobileDeviceFacet
 		owl:Class ,
 		sh:NodeShape
 		;
-	rdfs:subClassOf core:Facet ;
+	rdfs:subClassOf observable:DeviceFacet ;
```

Comment:
NACK here.  There is a pending design document from @sbarnum to explain why this is a subclass of `core:Facet`.  See [OC-97](https://unifiedcyberontology.atlassian.net/browse/OC-97) for reference.

Reply:
Agree. This should be subClassOf core:Facet